tough/src/editor/signed.rs

```diff
@@ -155,6 +156,26 @@ where
         let path = outdir.join(filename);
         std::fs::write(&path, &self.buffer).context(error::FileWrite { path })
     }
+
+    /// Append the old signatures for root role
+    pub fn add_old_signatures(mut self, old_signatures: Vec<Signature>) -> Result<Self> {
```

Comment:
Ideally we'd like to restrict this to the root role right?

Reply:
Yes. Because the rest of the metadata is not manually signed by us.